clusterloader2/pkg/measurement/common/scheduler_latency.go

```diff
@@ -96,7 +96,7 @@ func (s *schedulerLatencyMeasurement) Execute(config *measurement.Config) ([]mea
 		return nil, err
 	}
 
-	if !SSHToMasterSupported || !masterRegistered {
+	if !SSHToMasterSupported && !masterRegistered {
```

Comment:
didn't mention it before - but good catch, thanks!

Reply:
I signed it